policyengine_us/variables/gov/states/ar/tax/income/low_income/ar_low_income_tax_joint.py

```diff
@@ -10,7 +10,7 @@ class ar_low_income_tax_joint(Variable):
     unit = USD
     definition_period = YEAR
     reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=29"
-    defined_for = StateCode.AR
+    defined_for = "ar_low_income_tax_eligible"
 
     def formula(person, period, parameters):
```

Comment:
in the `ar_income_tax_before_non_refundable_credits_joint` var;

`return where(ar_low_income_tax_eligible, low_income_tax, total_main_rate)`

Reply:
do we just take either value of the two, rather than originally the smaller one?